src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs

```diff
@@ -408,11 +408,10 @@ public void SetSelectedPathOrNavigate(PathNavigationEventArgs e)
 				}
 			}
 
-			if (ParentShellPageInstance is null)
+			if (ParentShellPageInstance?.FilesystemViewModel?.WorkingDirectory is null)
```

Comment:
If you change it here, it might not work properly. I find line 402 suspicious. `FilesystemViewModel` might be null.
`PathNormalization.NormalizePath(s.FilesystemViewModel.WorkingDirectory) ==`

Reply:
Would you be willing to take over this fix? I've already tried twice 😂

Reply:
👍

Reply:
Done. https://github.com/files-community/Files/pull/13923